contribute/hacker-guide.md

```diff
@@ -177,7 +177,7 @@ Listening for transport dt_socket at address: 8002
 // skip docs for local publishing
 publishArtifact in (Compile, packageDoc) in ThisBuild := false
 // set version based on current sha, so that you can easily consume this build from another sbt project
-baseVersionSuffix := s"local-${Process("tools/get-scala-commit-sha").lines.head.substring(0, 7)}"
+baseVersionSuffix := s"local-${VersionUtil.gitProperties.value.sha}"
```

Comment:
Great, I'll make this change. Kind of bizarrely, I didn't need to fully qualify the scala.build.VersionUtil locally. Is that a convention thing or did it break in your environment?